src/lexer.mth

```diff
@@ -32,16 +32,17 @@ struct +Lexer {
     lexer-stack:List(Token)
     lexer-first-token:Token
     lexer-last-token:Token
-    +input: +Input
+    content: Str
+    index: UOffset
```

Comment:
should these be prefixed with `lexer-`?

Reply:
probably not, this PR is good, thank you!